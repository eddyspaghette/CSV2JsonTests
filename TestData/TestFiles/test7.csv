1,
2,101,201,301,401,501,601
3,102,202,302,402,502,602
4,103,203,303,403,503,603
5,104,204,304,404,504,604
6,105,205,305,405,505,605
7,106,206,306,406,506,606
8,107,207,307,407,507,607
9,108,208,308,408,508,608
10,109,209,309,409,509,609
11,110,210,310,410,510,610
12,111,211,311,411,511,611
13,112,212,312,412,512,612
14,113,213,313,413,513,613
15,114,214,314,414,514,614
