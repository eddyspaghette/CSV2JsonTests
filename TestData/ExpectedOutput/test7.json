[
 {
   "1": 2,
   "FIELD2": 101,
   "FIELD3": 201,
   "FIELD4": 301,
   "FIELD5": 401,
   "FIELD6": 501,
   "FIELD7": 601
 },
 {
   "1": 3,
   "FIELD2": 102,
   "FIELD3": 202,
   "FIELD4": 302,
   "FIELD5": 402,
   "FIELD6": 502,
   "FIELD7": 602
 },
 {
   "1": 4,
   "FIELD2": 103,
   "FIELD3": 203,
   "FIELD4": 303,
   "FIELD5": 403,
   "FIELD6": 503,
   "FIELD7": 603
 },
 {
   "1": 5,
   "FIELD2": 104,
   "FIELD3": 204,
   "FIELD4": 304,
   "FIELD5": 404,
   "FIELD6": 504,
   "FIELD7": 604
 },
 {
   "1": 6,
   "FIELD2": 105,
   "FIELD3": 205,
   "FIELD4": 305,
   "FIELD5": 405,
   "FIELD6": 505,
   "FIELD7": 605
 },
 {
   "1": 7,
   "FIELD2": 106,
   "FIELD3": 206,
   "FIELD4": 306,
   "FIELD5": 406,
   "FIELD6": 506,
   "FIELD7": 606
 },
 {
   "1": 8,
   "FIELD2": 107,
   "FIELD3": 207,
   "FIELD4": 307,
   "FIELD5": 407,
   "FIELD6": 507,
   "FIELD7": 607
 },
 {
   "1": 9,
   "FIELD2": 108,
   "FIELD3": 208,
   "FIELD4": 308,
   "FIELD5": 408,
   "FIELD6": 508,
   "FIELD7": 608
 },
 {
   "1": 10,
   "FIELD2": 109,
   "FIELD3": 209,
   "FIELD4": 309,
   "FIELD5": 409,
   "FIELD6": 509,
   "FIELD7": 609
 },
 {
   "1": 11,
   "FIELD2": 110,
   "FIELD3": 210,
   "FIELD4": 310,
   "FIELD5": 410,
   "FIELD6": 510,
   "FIELD7": 610
 },
 {
   "1": 12,
   "FIELD2": 111,
   "FIELD3": 211,
   "FIELD4": 311,
   "FIELD5": 411,
   "FIELD6": 511,
   "FIELD7": 611
 },
 {
   "1": 13,
   "FIELD2": 112,
   "FIELD3": 212,
   "FIELD4": 312,
   "FIELD5": 412,
   "FIELD6": 512,
   "FIELD7": 612
 },
 {
   "1": 14,
   "FIELD2": 113,
   "FIELD3": 213,
   "FIELD4": 313,
   "FIELD5": 413,
   "FIELD6": 513,
   "FIELD7": 613
 },
 {
   "1": 15,
   "FIELD2": 114,
   "FIELD3": 214,
   "FIELD4": 314,
   "FIELD5": 414,
   "FIELD6": 514,
   "FIELD7": 614
 }
]